h2o-docs/src/product/data-science/gam.rst

```diff
@@ -11,177 +11,222 @@ A generalized additive model (GAM) is a :ref:`glm` in which the linear predictor
+- **scale_tp_penalty_mat**: Scale penalty matrix for thin plate smoothers. Defaults to ``False``.
 
--  `link <algo-params/link.html>`__: Specify a link function (Identity, Family_Default, Logit, Log, Inverse, Tweedie, or Ologit). The default value is Family_Default.
+-  **keep_gam_cols**: Specify whether to save keys storing GAM columns. This option defaults to ``False`` (disabled).
 
-   -  If the family is **Gaussian**, then **Identity**, **Log**, and **Inverse** are supported.
-   -  If the family is **Binomial**, then **Logit** is supported.
-   -  If the family is **Fractionalbinomial**, then Logit is supported.
-   -  If the family is **Poisson**, then **Log** and **Identity** are supported.
-   -  If the family is **Gamma**, then **Inverse**, **Log**, and **Identity** are supported.
-   -  If the family is **Tweedie**, then only **Tweedie** is supported.
-   -  If the family is **Multinomial**, then only **Family_Default** is supported. (This defaults to ``multinomial``.)
-   -  If the family is **Quasibinomial**, then only **Logit** is supported.
-   -  If the family is **Ordinal**, then only **Ologit** is supported
-   -  If the family is **Negative Binomial**, then only **Log** and **Identity** are supported.
-   - If the family is **AUTO**,
+- `auc_type <algo-params/auc_type.html>`__: Set the default multinomial AUC type. Must be one of:
 
-      - and a link is not specified, then the link is determined as **Family_Default** (defaults to the family to which AUTO is determined).
-      - and a link is specified, the link is used so long as the specified link is compatible with the family to which AUTO is determined. Otherwise, an error message is thrown stating that AUTO for underlying data requires a different link and gives a list of possible compatible links.
-      - The list of supported links for ``family = AUTO`` is:
+   - ``"AUTO"`` (default)
+   - ``"NONE"``
+   - ``"MACRO_OVR"``
+   - ``"WEIGHTED_OVR"``
+   - ``"MACRO_OVO"``
+   - ``"WEIGHTED_OVO"``
 
-          1. If the response is **Enum** with cardinality = 2, then **Logit** is supported.
-          2. If the response is **Enum** with cardinality > 2, then only **Family_Default** is supported (this defaults to **multinomial**).
-          3. If the response is numeric (**Real** or **Int**), then **Identity**, **Log**, and **Inverse** are suported.
+Hyperparameters
+'''''''''''''''
 
--  `prior <algo-params/prior.html>`__: Specify prior probability for p(y==1). Use this parameter for logistic regression if the data has been sampled and the mean of response does not reflect reality. This value must be a value in the range (0,1) or set to -1 (disabled).  This option is set to -1 (disabled) by default.  
-   
-     **Note**: This is a simple method affecting only the intercept. You may want to use weights and offset for a better fit.
+These parameters can be used in grid search.
 
```

Comment:
Please also add the following parameters to the grid search

scale
num_knots
spline_order
bs

Reply:
All of these are all already under the grid search section :)